crates/interpreter/src/syntax.rs

```diff
@@ -546,6 +558,50 @@ impl CvtOp {
     }
 }
 
+#[cfg(feature = "threads")]
+macro_rules! impl_atomic_op {
+    ($u:ident) => {
+        paste::paste! {
+            impl AtomicOp {
+                pub fn [< $u >](&self, mem: *mut $u, c: $u) -> $u {
```

Comment:
nit: using paste seems unecessary here
```suggestion
                pub fn $u(&self, mem: *mut $u, c: $u) -> $u {
```
If/When moving this to the big `impl_op!` macro. I guess the function could be named `$n` instead.

Reply:
Removed usage of paste for this. See comment in other thread, `impl_op!` and `impl_op!` are called for different types.

Reply:
Indeed, it doesn't work.